ci/publish

```diff
@@ -45,7 +45,7 @@ if [ "${TRAVIS_TAG:-}" == "" ]; then
     #       jupyterhub/binderhub@123abcd which will be understood as a reference
     #       by GitHub.
     PR_OR_HASH=$(git log -1 --pretty=%B | head -n1 | sed 's/^.*\(#[0-9]*\).*/\1/' | sed 's/^\([0-9a-f]*\)-.*/@\1/')
-    EXTRA_MESSAGE="${TRAVIS_REPO_SLUG}${PR_OR_HASH}"
+    EXTRA_MESSAGE="${TRAVIS_REPO_SLUG}${PR_OR_HASH} ${LATEST_COMMIT_TITLE}"
```

Comment:
This environment variable isn't part of the CI system itself, but was previously named this even though it was mostly rendering to `#123` which was extracted from the latest commit title with `sed`.

The actual latest commit title can be extracted like this:

```
LATEST_COMMIT_TITLE=$(git log -1 --pretty=%B | head -n1)
```

I have made a mistake in #1197 I note also, which is that for the extraction of the commit hash, there also needs to be an inclusion of that (`%h`) for `sed` to work with.

```diff
-    PR_OR_HASH=$(git log -1 --pretty=%B | head -n1 | sed 's/^.*\(#[0-9]*\).*/\1/' | sed 's/^\([0-9a-f]*\)-.*/@\1/')
+    PR_OR_HASH=$(git log -1 --pretty=%h-%B | head -n1 | sed 's/^.*\(#[0-9]*\).*/\1/' | sed 's/^\([0-9a-f]*\)-.*/@\1/')
```

Reply:
Thanks! I added a line to set the commit title.